src/main/java/appeng/core/Registration.java

```diff
@@ -225,26 +201,13 @@ private void registerSpatialDimension()
 	private void registerCraftHandlers( final IRecipeHandlerRegistry registry )
 	{
 		registry.addNewSubItemResolver( new AEItemResolver() );
-
-		registry.addNewCraftHandler( "hccrusher", HCCrusher.class );
-		registry.addNewCraftHandler( "mekcrusher", MekCrusher.class );
-		registry.addNewCraftHandler( "mekechamber", MekEnrichment.class );
-		registry.addNewCraftHandler( "grind", Grind.class );
-		registry.addNewCraftHandler( "crusher", Crusher.class );
-		registry.addNewCraftHandler( "pulverizer", Pulverizer.class );
-		registry.addNewCraftHandler( "macerator", Macerator.class );
-
-		registry.addNewCraftHandler( "smelt", Smelt.class );
-		registry.addNewCraftHandler( "inscribe", Inscribe.class );
-		registry.addNewCraftHandler( "press", Press.class );
 	}
 
-	public void initialize( @Nonnull final FMLInitializationEvent event, @Nonnull final File recipeDirectory, @Nonnull final CustomRecipeConfig customRecipeConfig )
+	public void initialize( @Nonnull final FMLInitializationEvent event, @Nonnull final File recipeDirectory )
 	{
 		Preconditions.checkNotNull( event );
 		Preconditions.checkNotNull( recipeDirectory );
 		Preconditions.checkArgument( !recipeDirectory.isFile() );
-		Preconditions.checkNotNull( customRecipeConfig );
 
 		final Api api = Api.INSTANCE;
 		final IPartHelper partHelper = api.partHelper();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "partHelper". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "partHelper" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1481)